packages/emitter-framework/src/typescript/components/interface-declaration.tsx

```diff
@@ -70,13 +60,10 @@ export interface InterfaceExpressionProps extends ts.InterfaceExpressionProps {
 }
 
 export function InterfaceExpression({ type, children }: InterfaceExpressionProps) {
-  const members = type ? membersFromType(type) : [];
```

Comment:
Q: why is the guard here? The type of `type` suggests its always availabe?

Reply:
I don't know...

Reply:
yeah type should always be set according to the definition, proably just a mistake